tests/integration/basic-css-spec.js

```diff
@@ -95,7 +95,7 @@ test.describeIntegrationTest("Css Bundling:", function() {
 
 	    givenImages('an-image-there.jpg');
 
-        test.given.BundleOption("-rewriteimagefileroot:" + test.given.TestDirectory + " -rewriteimageoutputroot:combined");
+        test.given.CommandLineOption("-rewriteimagefileroot:" + test.given.TestDirectory + " -rewriteimageoutputroot:combined");

```

Comment:
I wanted to be able to specify options of the bundle level in tests, so I renamed this existing `Given` function to `CommandLineOption` since that's what it actually is used for.